packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -228,6 +231,30 @@ class _SelectableRegionState extends State<SelectableRegion> with TextSelectionD
     );
   }
 
+  @override
+  void didChangeDependencies() {
+    super.didChangeDependencies();
+    if (defaultTargetPlatform != TargetPlatform.iOS && defaultTargetPlatform != TargetPlatform.android) {
```

Comment:
Maybe use a switch case that we would catch it if a new platform is added to flutter